test/cmd/describer.sh

```diff
@@ -33,5 +33,16 @@ os::cmd::try_until_success 'eventnum=$(oc get events | wc -l) && [[ $eventnum -g
 # resources without describers get a default
 os::cmd::expect_success_and_text 'oc describe events' 'Namespace:	cmd-describer'
 
+# TemplateInstance
+os::cmd::expect_success 'oc create -f test/extended/testdata/templates/templateinstance_objectkinds.yaml'
+os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance' 'FOO:	3 bytes'
```

Comment:
running `os::cmd::expect_success_and_text 'oc describe templateinstances templateinstance'` many times seems a little clumsy, are there any better alternatives?

Reply:
I asked @bparees about this and he couldn't think of anything offhand.